Sources/CameraManager.swift

```
@@ -570,6 +570,8 @@ open class CameraManager: NSObject, AVCaptureFileOutputRecordingDelegate, UIGest
                     PHPhotoLibrary.requestAuthorization { status in
                         if status == PHAuthorizationStatus.authorized {
                             self._saveImageToLibrary(atFileURL: filePath, imageCompletion)
+                        } else {
+                            self._show(NSLocalizedString("Photos access denied", comment: ""), message: NSLocalizedString("Please allow \(Bundle.main.infoDictionary?[String(kCFBundleNameKey)] ?? "App") to access your photos in \"Settings -> Privacy -> Photos\"", comment: ""))
```

should throw an error here